contracts/src/v0.8/ccip/pools/GHO/UpgradeableTokenPool.sol

```diff
@@ -599,4 +598,6 @@ abstract contract UpgradeableTokenPool is IPoolV1, Ownable2StepMsgSender {
       }
     }
   }
+
+  uint256[42] private __gap;
```

Comment:
let's move it right at the end of the state variables so it's not forgotten

Reply:
it's mentioned in updates comments at the top of the contract, standard practice for this has always been to be hidden away at the bottom though, no?

Reply:
[59c3763](https://github.com/aave/ccip/pull/20/commits/59c3763bb5acc4391cae3018000932350198a584)